### DISCUSSION TITLE
[Community Call] November 15, 2022

### DISCUSSION THREAD

**Discussion by natemoo-re** [withastro/roadmap#390] [Announcements]
- **Announcement** A new Pre-Release version of the Astro VS Code extension is available! (0.29.0)
  - Added support for is:raw
  - Added support for props using the shorthand syntax (<element {prop} />)
  - Added support for using template literals directly for attributes
  - Added support for attributes using @ and dots in their name (this unlocks Alpine support)
  - Added support for getting file references for JavaScript files inside Astro files
  - Fixed colgroup element causing syntax errors
  - Fixed doctype element causing completions to not work when it wasn't self-closed
  - Fixed completions not working after script tags in certain cases
  - Fixed certain keywords with lesser-than comparisons causing syntax errors inside script tags
  - Fixed HTML comments causing completions to not work in certain cases
  - Fixed HTML comments inside strings inside expressions causing syntax errors
  - Fixed tags inside strings causing syntax errors
- **Docs**
  - Search Stats are being investigated
  - [Astro Tutorial](https://docs.astro.build/en/tutorial/0-introduction/)
  - **7 New Contributors**
- **RFC** https://github.com/withastro/rfcs/pull/373
- **RFC** https://github.com/withastro/rfcs/pull/381

_If there is something that you would like to see discussed, please comment below. We will prioritize items already on the agenda, but if there is time to spare we will open up the call to get attention on and discuss [other threads and feature requests](https://github.com/withastro/rfcs/discussions)._

**Comment by louiss0** [comment]
I would like to see #364 discussed. Astro does not provide a way for sets of HTML to be separated and then put back together.  `<AstroTemplateOutlet/>` solves this problem. By the way I ❤️ Astro Icon good job. I want Astro to have a native way of rendering the content that is created inside template tags.
